core/apply/endpoint.go

```diff
@@ -68,3 +97,4 @@ type depTokensResponse struct {
 
 func (r depTokensResponse) error() error { return r.Err }
 func (r blueprintResponse) error() error { return r.Err }
+func (r profileResponse) error() error   { return r.Err }
```

Comment:
do you just mean physically move the them in the file so they're grouped next to (line-wise) with the type delcaration?